src/content/RepoPage/RepoTable.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  DataTable,
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableExpandHeader,
+  TableHeader,
+  TableBody,
+  TableExpandRow,
+  TableCell,
+  TableExpandedRow,
+} from 'carbon-components-react';
+
+const RepoTable = ({ rows, headers }) => {
+  return (
+    <DataTable
+      rows={rows}
+      headers={headers}
+      render={({
+        rows,
+        headers,
+        getHeaderProps,
+        getRowProps,
+        getTableProps,
+      }) => (
+        <TableContainer
+          title="Carbon Repositories"
+          description="A collection of public Carbon repositories.">
+          <Table {...getTableProps()}>
+            <TableHead>
+              <TableRow>
+                <TableExpandHeader />
+                {headers.map(header => (
+                  <TableHeader {...getHeaderProps({ header })}>
+                    {header.header}
+                  </TableHeader>
+                ))}
+              </TableRow>
+            </TableHead>
```

Comment:
You're missing the Table Body component. See https://www.carbondesignsystem.com/tutorial/react-step-2#build-data-table